source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -559,6 +559,40 @@ def geojson():
     ]
 }
 
+BRAGI_MOCK_RESPONSE_WITHOUT_COMMENTS_IN_STOP_AREA = {
+    "type": "FeatureCollection",
+    "geocoding": {
+        "version": "0.1.0",
+        "query": ""
+    },
+    "features": [
+        {
+            "type": "Feature",
+            "geometry": {
+                "coordinates": [
+                    3.282103,
+                    49.847586
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "geocoding": {
+                    "city": "Bobtown",
+                    "housenumber": "20",
```

Comment:
this field does not exists with stop area

Reply:
I have cleaned the mock to be realistic
:white_check_mark: